.editorconfig

```diff
@@ -20,4 +20,4 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 
 [{*.py, *.pyi}]
-indent_size = 4
```

Comment:
What is the change here?

Reply:
probably 'no newline at EOF' or something similar